src/cpp/core/r_util/RToolsInfo.cpp

```diff
@@ -276,34 +276,50 @@ RToolsInfo::RToolsInfo(const std::string& name,
       // not actually be defined or available in Rtools
       clangArgs.push_back("-U_MSC_VER");
 
-      // set GNUC levels
-      // (required for _mingw.h, which otherwise tries to use incompatible MSVC defines)
-      clangArgs.push_back("-D__GNUC__=10");
-      clangArgs.push_back("-D__GNUC_MINOR__=3");
-      clangArgs.push_back("-D__GNUC_PATCHLEVEL__=0");
-
-      // get C headers paths
-      auto cStems = {
-         "x86_64-w64-mingw32.static.posix/lib/gcc/x86_64-w64-mingw32.static.posix/10.3.0/include",
-         "x86_64-w64-mingw32.static.posix/include",
-         "x86_64-w64-mingw32.static.posix/lib/gcc/x86_64-w64-mingw32.static.posix/10.3.0/include-fixed"
-      };
-
-      for (auto&& stem : cStems)
-         cIncludePaths.push_back(installPath.completeChildPath(stem).getAbsolutePath());
-
-      // get C++ headers
-      auto cppStems = {
-         "x86_64-w64-mingw32.static.posix/lib/gcc/x86_64-w64-mingw32.static.posix/10.3.0/include/c++",
-         "x86_64-w64-mingw32.static.posix/lib/gcc/x86_64-w64-mingw32.static.posix/10.3.0/include/c++/x86_64-w64-mingw32.static.posix",
-         "x86_64-w64-mingw32.static.posix/lib/gcc/x86_64-w64-mingw32.static.posix/10.3.0/include/c++/backward",
-         "x86_64-w64-mingw32.static.posix/lib/gcc/x86_64-w64-mingw32.static.posix/10.3.0/include",
-         "x86_64-w64-mingw32.static.posix/include",
-         "x86_64-w64-mingw32.static.posix/lib/gcc/x86_64-w64-mingw32.static.posix/10.3.0/include-fixed",
-      };
-
-      for (auto&& stem : cppStems)
-         cppIncludePaths.push_back(installPath.completeChildPath(stem).getAbsolutePath());
+      std::string toolDir = "x86_64-w64-mingw32.static.posix";
```

Comment:
We don't have any guarantee this directory will still be used in future builds of Rtools. (what if future builds aren't static? what if MinGW changes the name for some reason?)